encord/user_client.py

```diff
@@ -265,12 +270,30 @@ def create_project(
         dataset_hashes: List[str],
         project_description: str = "",
         ontology_hash: str = "",
-        source_projects: List = list(),
+        workflow_type: ProjectWorkflowType = ProjectWorkflowType.MANUAL_QA,
+        source_projects: List[str] = list(),
     ) -> str:
+        """
+        Creates a new project and returns its uid ('project_hash')
+
+        Args:
+            project_title: the title of the project
+            dataset_hashes: a list of the dataset uids that the project will use
+            project_description: the optional description of the project
+            ontology_hash: the uid of an ontology to be used. If omitted, a new empty ontology will be created
+            workflow_type: the type of the quality control workflow to use. Currently either `ProjectWorkflowType.MANUAL_QA`
+                of `ProjectWorkflowType.BENCHMARK_QA`
+            source_projects: only for Benchmark QA projects, a list of project ids (project_hash-es)
+                that contain the benchmark source data
+
+        Returns:
+            the uid of the project.
+        """
         project = {
             "title": project_title,
             "description": project_description,
             "dataset_hashes": dataset_hashes,
+            "workflow_type": workflow_type.value,
```

Comment:
I guess this could also be return on reading the project data, but probably not important for now.